### DISCUSSION TITLE
Debug menu should not show in the Release build

### DISCUSSION THREAD

**Discussion by omidkrad** [Dotnet-Boxed/Templates#29] [Q&A]
When you run the website as **Release**, the _Debug_ menu with _Runtime Information_ should not render.

**Comment by RehanSaeed** [comment]
This was true for MVC 5 but MVC 6 has changed a lot. You have to now use the environment variable to control the Development, Staging and Production environments. The Debug and Release config now just controls the building of DLL's and precompiling views.

**Comment by omidkrad** [comment]
Good to know. That makes sense for dnx/multi-platform. Thanks.
